src/features/bumpkins/components/DynamicNFT.tsx

```diff
@@ -24,15 +24,22 @@ export const DynamicNFT: React.FC<Props> = ({
   const parts = cloneDeep(bumpkinParts);
 
   useEffect(() => {
+    let isSubscribed = true;
     const load = async () => {
       const image = await buildImage({
         parts,
       });
 
-      setImageSrc(image);
+      if (isSubscribed) {
```

Comment:
What is this used for?

Reply:
To prevent setting state on an unmounted component if async task finishes late.